rasa/core/broker.py

```diff
@@ -56,21 +60,71 @@ def publish(self, event: Dict[Text, Any]) -> None:
         raise NotImplementedError("Event broker must implement the `publish` method.")
 
 
+def create_rabbitmq_ssl_options(
```

Comment:
optional (since it's a backport): can we reuse `create_ssl_context` from `rasa.server`?

Reply:
no, it uses "password" authentication

Reply:
couldn't it be the case that the rabbit certificate is also encrypted?

Reply:
oh yeah i forgot about that, there is definitely a possibility that there's a password

Reply:
👍 reusing server SSL context function now, with password auth if provided